packages/fxa-settings/src/lib/cache.ts

```diff
@@ -0,0 +1,54 @@
+import { InMemoryCache, makeVar, gql } from '@apollo/client';
+import Storage from './storage'
+
+const storage = Storage.factory('localStorage');
+
+export function getSessionToken(): string {
+  const storedAccounts = storage.get('accounts');
+  const currentAccountUid = storage.get('currentAccountUid');
+  // TODO: protect from if user doesn't have sessionToken (probably redirect them back to login)
+  return storedAccounts[currentAccountUid].sessionToken;
+}
+
+export function sessionToken(token: string) {
+  const storedAccounts = storage.get('accounts');
+  const currentAccountUid = storage.get('currentAccountUid');
+  storedAccounts[currentAccountUid].sessionToken = token;
+  storage.set('accounts', storedAccounts)
+}
+
+// The hierarchy may change but starting simple
+export const oldPassword = makeVar('')
+export const newPassword = makeVar('')
+
+export const typeDefs = gql`
+  extend type Query {
+    sessionToken: String!
+    oldPassword: String
+    newPassword: String
+  }
+`;
+
+export const cache = new InMemoryCache({
```

Comment:
most examples i see break out the cache into it's own file. here you'll see i've made a couple reactive variables to hold old/new password. I imagine we'll do so with other local/non-persistent state.